cdapython/Paginator.py

```diff
@@ -20,22 +20,23 @@
 
     from cdapython.results.result import Result
 
-TPaginator = TypeVar("TPaginator", bound="Paginator")
-
 
 class Paginator:
     def __init__(
-        self: TPaginator,
+        self: Self,
```

Comment:
It is just use for type checking it helps with auto complete the reason why I was using the type parameter was because of my pie was complaining about the self type https://docs.python.org/3/library/typing.html#typing.Self